voluncheer/voluncheer/settings.py

```diff
@@ -34,15 +34,20 @@
 #     "voluncheer-dev.us-west-2.elasticbeanstalk.com",
 # ]
 # Security
-CSRF_TRUSTED_ORIGINS = ["http://localhost", "http://127.0.0.1"]
+CSRF_TRUSTED_ORIGINS = [
+    "http://localhost",
+    "http://127.0.0.1",
+    "http://172.31.24.255",
```

Comment:
Can you add comments for these addresses explaining why they are here and what they refer to?

Reply:
Ohh, These IPs are for Email verification. They shouldn't be included here